hyrule/anaphoric.hy

```diff
@@ -75,10 +52,11 @@ variable name, as in ``(print \"My favorite Stephen King book is\" 'it)``."
        1
        2
        3"
-  (rit `(for [~it ~xs]
-    (when (not ~(R form))
-      (break))
-    ~@(R body))))
+  `(let [it None]
+    (for [it ~xs]
+      (when (not ~form)
+        (break))
+      ~@body)))
 
```

Comment:
wow core `let` really made these implementations very clean :rocket: :partying_face:

Reply:
Yep, it turns out that the way the new `let` works is perfect for anaphoric macros.